src/actions/chatActions.js

```diff
@@ -143,6 +149,49 @@ export const sendMessageByContactAction = (username: string, message: Object) =>
   };
 };
 
+export const getChatDraftByContactAction = (contactId: string) => {
+  return async (dispatch: Function) => {
+    try {
+      const { drafts } = await storage.get('chat');
+      const [chatDraft, chatDrafts] = partition(drafts, { contactId });
+      const { draftText = '' } = chatDraft[0] || {};
+
+      if (!draftText || draftText === '') { return; }
+
+      dispatch(saveDbAction('chat', { drafts: chatDrafts }, true));
+      dispatch({
+        type: ADD_CHAT_DRAFT,
+        payload: { draftText },
+      });
+    } catch (e) {
+      console.log(e); // eslint-disable-line
+    }
+  };
+};
+
+export const clearChatDraftStateAction = () => {
+  return async (dispatch: Function) => {
+    dispatch({
+      type: CLEAR_CHAT_DRAFT,
+    });
+  };
+};
+
+export const saveDraftAction = (contactId: string, draftText: string) => {
+  return async (dispatch: Function) => {
+    try {
```

Comment:
I'm wondering if Sentry would be able to catch the error when we wrapped everything into try..catch?